src/Input.js

```diff
@@ -0,0 +1,14 @@
+import { Console } from '@woowacourse/mission-utils';
+import { INPUT_MESSAGE } from './constants/message.js';
+import { ERROR_MESSAGE } from './constants/error.js';
+
+export default class Input {
+  async getInput() {
+    try {
+      const input = await Console.readLineAsync(`${INPUT_MESSAGE}`);
```

Comment:
```suggestion
      const input = await Console.readLineAsync(INPUT_MESSAGE);
```
이렇게 작성하지 않고, 템플릿 리터럴을 사용하신 이유가 있으신가요?
그리구 `getInput`의 파라미터로 `INPUT_MESSAGE`를 받는 것은 어떻게 생각하시는지 궁금합니다! 그럼 더 재사용성이 높아지지 않을까 해서요 :)

Reply:
좋은 말씀과 팁 주셔서 감사합니다.

Reply:
리팩토링 하기 전, 처음에 문자열을 조합하여 INPUT_MESSAGE를 함께 사용하기 위해서 템플릿 리터럴을 사용하다가 리팩토링 이후 수정하였는데, 미처 저 부분을 빼먹은 것 같습니다. 조언 감사합니다!

Reply:
파라미터로 input 받아서 재사용성이 높아지는 것은 좋은 아이디어 인 것 같습니다! 다음 과제때 고려해보겠습니다.